wgengine/capture/ts-dissector.lua

```diff
@@ -0,0 +1,27 @@
+tsdebug_ll = Proto("tsdebug", "Tailscale debug")
+PATH = ProtoField.string("tsdebug.PATH","PATH", base.ASCII)
+tsdebug_ll.fields = {PATH}
+
+function tsdebug_ll.dissector(buffer, pinfo, tree)
+    pinfo.cols.protocol = tsdebug_ll.name
+    packet_length = buffer:len()
+    local offset = 0
+    local subtree = tree:add(tsdebug_ll, buffer(), "Tailscale packet")
+
+    -- -- Get path UINT16
+    local path_id = buffer:range(offset, 2):le_uint()
+    if     path_id == 0 then subtree:add(PATH, "FromLocal")
+    elseif path_id == 1 then subtree:add(PATH, "FromPeer")
```

Comment:
can there be a separate path for "FromPeerDropped" vs "FromPeerAccepted" or something?

Reply:
I'd rather add additional metadata (maybe an enum `acl_verdict` with pre-intercept, accept, drop) than overload `path`. We have total flexibility here, Wireshark is really nice like that.

Reply:
Sure, that works.